homework/task-1.js

```diff
@@ -0,0 +1,5 @@
+
+
+for(var line ="#"; line.length <=7;line +="#"){

```

Comment:
```var``` -ზე ხომ შევთანხმდით რომ არ უნდა გამოიყენოთ;
სფეისებს მიაქციე ყურადღება!
```suggestion
for(let line ="#"; line.length <=7; line +="#") {

```